components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -62,13 +65,16 @@ const TrialBenefitsModal: React.FC<Props> = (props: Props): JSX.Element | null =
     };
 
     const redirectToConsolePage = (route: string) => {
+        redirectTo(route, route.split('/').pop() ?? 'unknown');
```

Comment:
nit: This was easier to understand when we explicitly named the `lastSection` variable.